CreeDictionary/API/models.py

```diff
@@ -389,89 +369,91 @@ def fetch_lemma_by_user_query(user_query: str) -> "CreeAndEnglish":
             ):
                 english_results.add(
                     EnglishResult(
-                        MatchedEnglish(user_query),
-                        NormatizedCree(wordform.text),
-                        Lemma(wordform),
+                        MatchedEnglish(user_query), wordform, Lemma(wordform),
                     )
                 )  # will become  (user_query, inflection.text, wordform)
 
         return CreeAndEnglish(cree_results, english_results)
 
-    @staticmethod
-    def search(user_query: str) -> Tuple[str, List[SearchResult]]:
-        """
-        Returns the matched language (as an ISO 639 code), and list of search
-        results.
-        """
-        cree_results, english_results = Wordform.fetch_lemma_by_user_query(user_query)
+    def __lt__(self, other):
```

Comment:
Might consider using [`functools.total_ordering`](https://docs.python.org/3.6/library/functools.html#functools.total_ordering), but I'm not sure how that works inside a Django `Model`.

Reply:
ooOOo I think this `__lt__` is old code I added and it's not needed any more. I'll clean it!